server/main_test.go

```
@@ -0,0 +1,65 @@
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/pocketbase/pocketbase/core"
+	"github.com/pocketbase/pocketbase/tools/router"
+)
+
+func TestHealthz(t *testing.T) {
+	request := httptest.NewRequest(http.MethodGet, "/api/healthz", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)